solr/solr-ref-guide/modules/deployment-guide/pages/taking-solr-to-production.adoc

```diff
@@ -455,7 +455,7 @@ Ideally this would mean more machines, but budget constraints might make that im
 There is another issue once the heap reaches 32GB.
 Below 32GB, Java is able to use compressed pointers, but above that point, larger pointers are required, which uses more memory and slows down the JVM.
```

Comment:
Add references here to what this means. For example:
* https://blog.codecentric.de/35gb-heap-less-32gb-java-jvm-memory-oddities
* https://blog.foundev.pro/2021/02/03/heap-size-around-31gb.html
* https://www.baeldung.com/jvm-compressed-oops
* https://docs.oracle.com/javase/7/docs/technotes/guides/vm/performance-enhancements-7.html#compressedOop